_site/404.html

```diff
@@ -178,7 +181,7 @@ <h6>Find more Red Hat Middleware Community Projects</h6>
           
             <li><a href="http://teiid.jboss.org/" target="_blank">Teiid</a></li>
           
-            <li><a href="http://wildfly-swarm.io/" target="_blank">WildFly Swarm</a></li>
+            <li><a href="https://thorntail.io/posts/the-end-of-an-era/" target="_blank">WildFly Swarm</a></li>
```

Comment:
i think we need to change here also Wildfly Swarm to Thorntail, otherwise it is confusing.
```suggestion
            <li><a href="https://thorntail.io/posts/the-end-of-an-era/" target="_blank">Thorntail</a></li>
```